CMakeLists.txt

```diff
@@ -62,6 +62,11 @@ option(
   VELOX_BUILD_MINIMAL
   "Build a minimal set of components only. This will override other build options."
   OFF)
+option(
```

Comment:
nit: Should we add a makefile target that exercises this ?

Reply:
Good idea. I'll add one.